### PR TITLE
Update CONTRIBUTING.md with release instructions.

### PR DESCRIPTION
**Pull request by troessner**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -190,3 +190,21 @@ We are following [semantic versioning](http://semver.org/).
 If you're working on a change that is breaking backwards-compatibility
 just go ahead with your pull request like normal. We'll discuss this then in
 the pull request and help you to point your pull request to the right branch.
+
+### Releasing Reek
+
+* In this example we assume the current version is 5.3.1 and you want to update to 5.3.2
```
Comment (mvz) [inline CONTRIBUTING.md]: This should probably not have a bullet.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -190,3 +190,21 @@ We are following [semantic versioning](http://semver.org/).
...
+* Create a branch with a name like "prepare-v.5.3.2"
+* Update the version in `lib/reek/version.rb`
+* List all relevant changes in `CHANGELOG.md`
+* Update the version number in our cucumber features, otherwise the build will fail. You can do this quite easily via
+
+```Bash
```
Comment (mvz) [inline CONTRIBUTING.md]: This should be indented two spaces so it is rendered inside the `<li>`. You may have to remove the empty line 201 as well.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -190,3 +190,21 @@ We are following [semantic versioning](http://semver.org/).
...
+find features/ -type f -exec sed -i '' 's/v5.3.1/v5.3.2/g' {} +
+```
+* Push the branch, create a pull request, have it reviewed and merged
+* Pull the latest master and then do a
+
+```Bash
```
Comment (mvz) [inline CONTRIBUTING.md]: Same as the previous code block.

**Comment by troessner** [comment]
Addressed your feedback @mvz ;)

**Pull request merged**